app-misc/gnutu/gnutu-2.9.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="GNU Student's Timetable for polish users"
+HOMEPAGE="https://sourceforge.net/projects/gnutu/"
+SRC_URI="https://master.dl.sourceforge.net/project/gnutu/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-lang/mono-1.2.5.1-r1
+	>=dev-dotnet/gtk-sharp-2.12.21
+"
+BEPEND="${RDEPEND}
+	sys-devel/gettext
+"
+
+src_prepare() {
+	# Remove deprecated Application tag from .desktop file
```

Comment:
Ideally you'd report/fix this upstream, and add an issue link here.

Reply:
@juippis Thanks, yes, I know that is the best solution, though the upstream doesn't seem to be very active with this project, I didn't see any activity since the beggining of 2019. Nevertheless, I reported the issue to the upstream now.